worker/wrangler.toml

```diff
@@ -1,16 +0,0 @@
-# Cloudflare Worker configuration
-account_id = "895762025d37fc687ecd72d7cc80204a"
-zone_id = "c192cf8ac042c681023493c52edd44c8"
-
-# shared
-type = "javascript"
-
-# default dev environment
-name = "cdn-dev"
-workers_dev = true
-
-# environment configuration
-[env.production]
-name = "cdn"
-route = "cdn.deno.land/*"
-workers_dev = false
```

Comment:
So terraform takes care of updating the worker?

Reply:
yes